src/app/dashboard/Missions/CreateMission.js

```diff
@@ -0,0 +1,482 @@
+    } catch (error) {
+      console.log(error);
+    }
+  }
+  // Functions for changing the states
+  const handleChangeMissionType = (event) => {
+    setMissionType(event.target.value);
+  };
+
+  const handleChangeFund = (event) => {
+    setMissionFund(event.target.value);
+  };
+
+  const handleRecipient = (event) => {
+    setRecipient({ ...recipient, [event.target.name]: event.target.value });
+  };
+
+  const handleVolunteer = (event) => {
+    setVolunteer({ ...volunteer, [event.target.name]: event.target.value });
+  };
+
+  const handleDrop = (event) => {
+    setDropOff({ ...dropOff, [event.target.name]: event.target.value });
+  };
+
+  const handlePickUp = (event) => {
+    setPickUp({ ...pickUp, [event.target.name]: event.target.value });
+  };
+
+  const handleComment = (event) => {
+    setComment(event.target.value);
+  };
+
+  const handleItems = (event) => {
+    setItems({ ...items, [event.target.name]: event.target.value });
+    console.log(items);
+  };
+
+  function addToFBCount() {
+    setFoodBoxArray([...foodBoxArray, foodBoxArray.length]);
+  }
+
+  return (
+    <Container maxWidth="sm">
+      <Typography className={classes.addMissionContainer} component="div">
+        <div className={classes.title}>
+          <AnnouncementIcon className={classes.icon} />
+          <Typography className={classes.titleHeader}>Add New Mission</Typography>
+        </div>
+        <Divider variant="middle" />
+        <Typography className={classes.inputHeader}>Mission Type</Typography>
+        <FormControl className={classes.form}>
+          <InputLabel
+            variant="outlined"
+            id="missionType"
+            shrink={true}
+            className={classes.inputLabel}
+          >
+            Select Mission Type
+          </InputLabel>
+          <Select
+            labelid="missionType"
+            variant="outlined"
+            className={classes.select}
+            value={missionType}
+            onChange={handleChangeMissionType}
+          >
+            <MenuItem className={classes.menuItem} value="food-box">
+              Food box
+            </MenuItem>
+          </Select>
+        </FormControl>
+        {missionType !== null ? (
+          <>
+            <Typography className={classes.inputHeader}>Select Food Box Options</Typography>
+            <FormControl className={classes.form}>
+              {Object.keys(foodBoxArray).map((boxNumber) => {
+                const { amount, name } = foodBoxArray[boxNumber];
+                return (
+                  <div className={classes.sideBySideSelect} key={boxNumber}>
+                    <Select
+                      onChange={handleItems}
+                      name={"item for box " + boxNumber}
+                      variant="outlined"
+                      className={classes.select}
+                      value={name}
+                    >
+                      <MenuItem value="Fruits and Veggies">Fruits & Veggies</MenuItem>
```

Comment:
@thirdeyeclub  Please populate this dropdown dynamically with the different types of Foodboxes that are offered by the organization.  We can't hardcode the values here.

Reply:
When I wrote this they did not exist at all yet.